src/Persistence/IsProxy.php

```diff
@@ -114,14 +115,19 @@ public function _repository(): ProxyRepositoryDecorator
 
     private function _autoRefresh(): void
     {
-        if (!$this->_autoRefresh) {
+        // "??=" fixes in ProxyHelper where object gets created without any props initialized
+        if (!($this->_autoRefresh ??= true)) {
```

Comment:
since `symfony/var-exporter` 7.1, there is a strange bug where `IsProxy::$_autorefresh` is not initialized, resulting in a stranger bug:
```
TypeError: Value of type null returned from ZenstruckFoundryTestsFixtureEntityGenericEntityProxy::__get() must be compatible with unset property ZenstruckFoundryTestsFixtureEntityGenericEntityProxy::$_autoRefresh of type bool
```

what's strange is that it seems there is not difference between `symfony/var-exporter` 7.0.8 and 7.1.0 :shrug: